src/wallet.py

```diff
@@ -0,0 +1,86 @@
+import json
+from json.decoder import JSONDecodeError
+
+import structlog
+from nacl.encoding import HexEncoder
+from nacl.signing import SigningKey
+import base64
+from cryptography.fernet import Fernet
+
+
+logger = structlog.get_logger(__name__)
+
+class Wallet:
+    def __init__(self): 
+        self.privatekey = ""
+        self.publickey = ""
+    
+    def generate_wallet():
+        password = input("\nSe necesita una contraseña, escriba una: ")   #Proposing a change 
```

Comment:
If we provide `wallet_pass` we no longer need this :)

Reply:
Included